cgroups/src/common.rs

```diff
@@ -355,3 +356,21 @@ pub(crate) fn default_devices() -> Vec<LinuxDevice> {
         },
     ]
 }
+
+pub(crate) fn delete_with_retry<P: AsRef<Path>>(path: P) -> Result<()> {
```

Comment:
I think it is more versatile to have the number of retries and the delay as arguments. What do you think?